runelite-client/src/main/java/net/runelite/client/game/ItemManager.java

```diff
@@ -281,6 +299,18 @@ public int getItemPrice(int itemID)
 		{
 			return 1000;
 		}
+		if (itemID == ItemID.MASTER_SCROLL_BOOK)
+		{
+			int totalPrice = 0;
+
+			for (Map.Entry<Varbits, Integer> entry : VARBIT_MASTERBOOK_ITEMIDMAP.entrySet())
+			{
+				final int amount = client.getVar(entry.getKey());
+				totalPrice += getItemPrice(entry.getValue()) * amount;
+			}
+
+			return totalPrice;
```

Comment:
Shouldn't you also add the price of the scroll book here?

Reply:
Yes, but not in this method as that would make getting correct price for multiple scroll books impossible